composer.json

```diff
@@ -3,14 +3,33 @@
     "description": "Ibexa Commerce eContent data installer",
     "minimum-stability": "stable",
     "license": "proprietary",
-    "type": "ibexa-bundle",
+    "type": "symfony-bundle",
```

Comment:
This allows us to take an advantage of Symfony Flex which will auto-enable bundle during `composer require`. It makes installation instruction shorter.